requirements/dev.txt

```diff
@@ -8,7 +8,6 @@ pytest-cov==4.1.0
 # mypy adds new rules in new minor versions, which could cause our PR check to fail
 # here we fix its version and upgrade it manually in the future
 mypy==1.3.0
-boto3-stubs[apigateway,cloudformation,ecr,iam,lambda,s3,schemas,secretsmanager,signer,stepfunctions,sts,xray]==1.28.7
```

Comment:
not an immediate issue but we should think about it - I wonder if we should move all typing stubs to base.txt, considering that the same could happen for any typing stubs we are using. That also yield another questions:
- How do we test if we've included all dependencies used for type hints into base.txt?
- Should we consider striping all type hints as part of release process after passing type checks?

Reply:
I think that's a good callout and both are valid questions worth thinking.